ruby/Makefile

```diff
@@ -0,0 +1,16 @@
+VERSION=2.6.3
+DOCKER_REPO=pubnative/ruby
+DOCKER_REPO_VERSION=$(DOCKER_REPO):$(VERSION)
+DOCKER_REPO_LATEST=$(DOCKER_REPO):latest
+
+build:
+	docker build --build-arg RUBY_VERION=${VERSION} -t $(DOCKER_REPO_VERSION) .
+	docker tag $(DOCKER_REPO_VERSION) $(DOCKER_REPO_LATEST)
+
+publish: publish-latest publish-version
+
+publish-latest: build
+    docker push $(DOCKER_REPO_LATEST)
```

Comment:
I’m pretty sure you need a tab here.

Reply:
@ValentinMouret I think it is tab indeed.